mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/Image.php

```diff
@@ -44,8 +44,10 @@ private function applyRoundedStyle(string $blockContent, array $parsedBlock): st
    */
   private function addImageSizeWhenMissing(array $parsedBlock, string $imageUrl): array {
     if (!isset($parsedBlock['attrs']['width'])) {
+      $maxWidth = $parsedBlock['email_attrs']['width'] ?? '660px';
```

Comment:
Please the constant for max-width https://github.com/mailpoet/mailpoet/blob/9b05e46690739349fcbd1db0f9061fa6bed3444f/mailpoet/lib/EmailEditor/Engine/SettingsController.php#L39
In the future, we will probably make the max width configurable, so looking for usage of the constant will be easier.